src/modules/jcms_rest/src/Plugin/rest/resource/AbstractRestResourceBase.php

```diff
@@ -426,8 +426,8 @@ protected function getDigestSnippet(Node $node) {
   protected function subjectsFromArticles(array $articles = NULL) : array {
     $subjects = [];
     foreach ($articles as $article) {
-      if (property_exists($article, 'subjects') && !empty($article->subjects)) {
-        foreach ($article->subjects as $subject) {
+      if (!empty($article['subjects'])) {
+        foreach ($article['subjects'] as $subject) {
```

Comment:
This is a bug that was discovered in implementation that has existed for a while. I will open another PR for this and will ensure that all affected content gets queued for reindexing.